faq/index.md

```diff
@@ -2,153 +2,280 @@
+## <a name="neighbors"></a>I have neighbors who would also like to join NYC Mesh. Can we get connected together?
 
-## <a name="nsm5-indoor"></a>I want a NanoStation but I'm not ready for a rooftop install
+Yes! We prioritize installs for entire buildings and set up high-bandwidth point-to-point connections with dedicated hardware. If your building would like this dedicated service please email us, reach out to us on [slack](https://slack.nycmesh.net/), send us an email at install@nycmesh.net or call us directly at (646) 727-0866. Using this service can replace your current ISP for very large buildings.
 
-It is easy to use the NanoStation indoors. It is 5GHz router, and all modern phones and most laptops support that. It looks a little strange but has a very good signal. If you have a long apartment or thick walls, a NanoStation is ideal.
+## <a name="id"></a>Do I need to provide identification to join NYC Mesh?
 
-## <a name="window"></a>I can see another node from my window. Can I put a router there?
+NYC Mesh asks for only basic contact information to be able to carry out the install. We welcome all New Yorkers to join the community regardless of immigration status or life circumstance. If you have concerns about privacy, please send us an email at help@nycmesh.net or talk to us in person at our monthly [meetup](https://www.meetup.com/nycmesh/).
 
-Yes! You can use a directional NanoStation in your window. It will work from behind glass if you are unable to place it outside the window. The simplest install is to velcro the flat surface of the NanoStation NSM5 to the window. We have a [blog post about NSM5 installs.](../blog/nsm5-install/)
+## <a name="slack"></a>What is Slack and how do I use it?
 
-## <a name="mobile"></a>Why don't we mesh mobile phones instead of routers?
+Slack is the main communication and collaboration platform we use at NYC Mesh. It consists of public channels, which anyone can post messages to, and invitation-only channels, which serve specific community groups and volunteer teams. Go to the [NYC Mesh Slack page](https://slack.nycmesh.net/) to sign up for an account and join the conversation!
 
-There is "mesh" software for mobile phones such as Serval and FireChat. This is mainly used for chat messages. It stores the messages until a phone is within range and then transfers the data. Mobile "mesh" networks are very low bandwidth, not real-time and require many phones in a small area. They are great for events or some emergency situations.
+# Getting connected to the mesh
 
-## <a name="secure"></a>Is the mesh secure?
+## <a name="covered"></a>How can I tell if I’m in the coverage zone for NYC Mesh?
 
-If you use the internet while on the mesh, you should use standard precautions such as using https (lock icon) web sites for secure connections, or use a VPN service.
+Connecting to the Internet with NYC Mesh requires a direct line of sight to an existing node. Look at our [map](../map) to see if you’re either:
```

Comment:
We should clarify that this is mostly applicable to supernodes/hubs not other end user nodes.

Reply:
Jon, could you have a go at directly editing this?

Reply:
I think someone directly involved with the planning of servicing installs should chime in.

Reply:
@zgiles I noticed you talking about node to node installs in \#bronx – can you offer an explanation of when node to node infrastructure gets built (vs. hub/spoke?) and set FAQ reader  expectations around this?

Reply:
We have ( and should encourage ) node to node installs all over the map, this is a primary feature of a mesh. I couldn't word this as "Internet", but rather talk about the The Coverage. Perhaps something along the lines of: "Connecting to the bulk of the mesh requires line of sight access to an existing node that is part of the mesh.. see our map.. etc. If you don't have line of sight to an existing node, don't worry, you can start your neighborhood. Find a few neighbors that are interested to connect to each other, share internet to start, connecting back to the main network over the internet. Local services in your neighborhood will work, and as the mesh grows to your area and you grow towards it, they will interconnect." ( Paragraphs / chop as needed )